### PR TITLE
modify tempk 1

### PR DESCRIPTION
**Pull request by kirantemp1**
pull request 1

### CODE REVIEW

**Comment by kirankgpro** [comment]
@kirant3  hv u discussed before

**Comment by kirantemp1** [comment]
@kirankgpro  yes iscussed

**Comment by kirankgpro** [comment]
ok

File: sample.txt [hunk 1]
```diff
@@ -1,18 +1 @@
-sample text
-
-
-second sample text
```
Comment (kirankgpro) [inline sample.txt:4]: why this 4 lines removed

Comment (kirantemp1) [reply sample.txt]: it was not necessary

File: sample.txt [hunk 2]
```diff
@@ -1,18 +1 @@
...
-
-ok
-
```
Comment (kirankgpro) [inline sample.txt:7]: 5 to 7 y removed

Comment (kirantemp1) [reply sample.txt]: its already discussed

File: sample.txt [hunk 3]
```diff
@@ -1,18 +1 @@
...
-
-creating a secod branch
```
Comment (kirankgpro) [inline sample.txt:9]: 9th line wt is this

File: sample.txt [hunk 4]
```diff
@@ -1,18 +1 @@
...
-
-
-
-creting merge conflict
-
-testing again
```
Comment (kirankgpro) [inline sample.txt:15]: 13 to 15 y removed

File: sample.txt [hunk 5]
```diff
@@ -1,18 +1 @@
...
-
-2nd test update done
```
Comment (kirankgpro) [inline sample.txt:17]: is it removed 17

File: sample.txt (hunk 4 above)
Comment (kirantemp1) [reply sample.txt]: yes removed its needed to be removed

Comment (kirankgpro) [reply sample.txt:15]: ok

File: sample.txt (hunk 1 above)
Comment (kirankgpro) [inline sample.txt:4]: what is this

File: sample.txt (hunk 3 above)
Comment (kirankgpro) [inline sample.txt:9]: why line 9

**Pull request merged**